### DISCUSSION TITLE
Google Season of Docs 2023

### DISCUSSION THREAD

**Discussion by Superskyyy** [apache/skywalking#10335] [Ideas]
https://developers.google.com/season-of-docs/docs/timeline  This is an event where Google will provide stipend (5000-15000usd!) for the project to hire some technical writers so to enhance our documentation. 

Is this something we can participate? If so I will send some extra information to maillist for everyone to check with and discuss how to proceed.

**Comment by Superskyyy** [comment]
https://developers.google.com/season-of-docs/docs/admin-guide 

Some Apache projects was selected in previous years. 
Gora: https://cwiki.apache.org/confluence/display/GORA/Google+Season+of+Docs+2021
Flink: https://flink.apache.org/news/2020/05/04/season-of-docs.html https://developers.google.com/season-of-docs/docs/2020/participants/project-apacheflink-haseeb1431

**Comment by wu-sheng** [comment]
I can't find any ASF-level discussion about this event. 
If you want to represent a project to submit this, you just need a mail list discussion/vote. If there is no concern(I assume there will not), you have been granted to do further.

**Reply by Superskyyy** [reply]
Sure, I will proceed on this as it's quite a good opportunity.
